solutions/remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -407,6 +413,63 @@
                 "description": "Diagnostics service endpoint url"
             },
             "defaultValue": "Undefined"
+        },
+        "telemetryStorageType": {
+            "type": "string",
+            "defaultValue": "tsi",
+            "allowedValues": [
+                "tsi", "cosmosdb"
+            ],
+            "metadata": {
+                "description": "Storage type for telemetry messages, supports TSI and CosmosDb"
+            }
+        },
+        "tsiEnvironmentName": {
+            "type": "string",
+            "defaultValue": "[concat('tsi-', take(uniqueString(subscription().subscriptionId, resourceGroup().id, parameters('solutionName')), 5))]",
+            "metadata": {
+                "description": "TSI environment name"
+            }
+        },
+        "tsiEnvironmentSkuName": {
+            "type": "string",
+            "defaultValue": "S1",
+            "allowedValues": [
+                "S1", "S2"
```

Comment:
nit: new lines per value

Reply:
yep. but I saw the other allowed values are using one line such as iotHubSku, iotHubTier. Just to keep them consistent.